app/models/booking.rb

```diff
@@ -0,0 +1,7 @@
+class Booking < ApplicationRecord
+  validates :check_in, presence: true, uniqueness: true
+  validates :check_out, presence: true, uniqueness: true
```

Comment:
What about validating the parameters that are not set by the user but by the controller method?
We still need to foolproof our creation process so that no booking is created with missing values (even if the missing value is not the user's fault but the developer's)
```suggestion
  validates :check_out, presence: true, uniqueness: true
  validates :space_id, presence: true
  validates :user_id, presence: true
```